pyproject.toml

```diff
@@ -33,7 +33,11 @@ select = ["E", "W", "F", "C", "N", "R", "D", "H"]
 # Ignore D107 Missing docstring in __init__
 ignore = ["W503", "E501", "D107"]
 # D100, D101, D102, D103: Ignore missing docstrings in tests
-per-file-ignores = ["tests/*:D100,D101,D102,D103,D104,D205,D212,D415,DCO020,DCO030"]
+per-file-ignores = [
+    "tests/*:D100,D101,D102,D103,D104,D205,D212,D415,DCO020,DCO030,DCO050",
+    "src/webserver.py:DCO060", # false alarms for classes with property functions
```

Comment:
What is the reason for this? Do you think we should document properties in the docstring?

Reply:
I added the `Attrs` section to the class docstring, this ignore has been removed, thanks!